nuxeo-drive-core/src/test/java/org/nuxeo/drive/fixtures/DefaultFileSystemItemFactoryFixture.java

```diff
@@ -695,10 +691,10 @@ public void testFileItem() throws Exception {
             assertEquals("File renamed by Joe.txt", updatedBlob.getFilename());
             // Check versioning => should be versioned since updated by a
             // different contributor
-            assertVersion("0.4", file);
+            assertVersion("0.5", file);
             fileVersions = session.getVersions(file.getRef());
-            assertEquals(4, fileVersions.size());
-            lastFileVersion = fileVersions.get(3);
+            assertEquals(5, fileVersions.size());
+            lastFileVersion = fileVersions.get(4);
             updatedBlob = (Blob) lastFileVersion.getPropertyValue("file:content");
             assertEquals("Renamed again.txt", updatedBlob.getFilename());
         }
```

Comment:
Remove "in {@link DefaultFileSystemItemFactory#needsVersioning(DocumentModel)}" in `ensureJustModified`